content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -801,7 +801,12 @@ This is commonly used by add-on API servers for unified authentication and autho
 <td>None</td>
 <td>Allows access to the resources required by most <a href="/docs/concepts/storage/persistent-volumes/#provisioner">dynamic volume provisioners</a>.</td>
 </tr>
-<tbody>
+<tr>
+<td><b>system:monitoring</b></td>
+<td><b>system:monitoring</b> group</td>
```

Comment:
Should group appear in bold?
```
<b>group</b>
```

I think the `<tt>` element is deprecated?
For code or endpoints, you can use backticks:
`/healthz`

The table data content could be formatted as:

Allows read access to control-plane monitoring endpoints such as the {{< glossary_tooltip term_id="kube-apiserver" text="kube-apiserver" >}} liveness and readiness endpoints (`/healthz`, `/livez`, `/readyz`), the individual health-check endpoints (`/healthz/*`, `/livez/*`, `/readyz/*`),  and `/metrics`. Note that individual health check endpoints and the metric endpoint may expose sensitive information.

Reply:
Hi @kbhawkey, using backticks resulted in the following (https://github.com/kubernetes/website/pull/22715#issuecomment-705832853) so I suggested to change to <tt>. It doesn't seem to be deprecated as the page renders as expected (https://5f7f97dbc52f3c00071d9bee--kubernetes-io-vnext-staging.netlify.app/docs/reference/access-authn-authz/rbac/). PTAL when you can, thank you! 

@logicalhan pinging to review Karen's suggestions! :D

Reply:
OK, the HTML table looks good.
/lgtm